lib/rules/value-no-vendor-prefix/README.md

```diff
@@ -75,11 +75,6 @@ a { cursor: -webkit-grab; }
 a { list-style-type: -moz-hangul; }
 ```
 
-<!-- prettier-ignore -->
-```css
-a { list-style-type: -moz-hangul-consonant; }
```

Comment:
This was technically true that it was being ignored but it was misleading.
i.e. it was ignored because it wasn't part of the variables that are considered unprefixable
see https://github.com/stylelint/stylelint/pull/7654/files#diff-5f9be0b5e6e40c3ad07fc3db2a8895c25da53046e10664c0112bbb9e237893c5R295